PaddleCV/image_classification/eval.py

```diff
@@ -93,10 +94,9 @@ def eval(args):
     exe = fluid.Executor(place)
     exe.run(fluid.default_startup_program())
 
-
     fluid.io.load_persistables(exe, pretrained_model)
 
-    val_reader = paddle.batch(reader.val(settings=args), batch_size=args.batch_size)
+    val_reader = reader.val(settings=args, batch_size=args.batch_size)
```

Comment:
batch_size has already passed in ```settings=args```

Reply:
https://github.com/PaddlePaddle/models/blob/c1f846dc358d140f8eafbd663514853856efe42e/PaddleCV/image_classification/reader_cv2.py#L301
function has a batch_size parameter